File: Feitos em C++ Parte 2/2923.cpp
#include<bits/stdc++.h>
using namespace std;
int main() {
	int t,a,w,c;
	while(cin>>t>>a>>w>>c){
		double veri = (a*100)/t;
		if(veri>=w&&veri<c){
			cout<<"warning"<<endl;
		}
		else if(veri>=c){
			cout<<"critical"<<endl;
		}
		else{
			cout<<"OK"<<endl;
		}
	}
	return 0;
}

File: Feitos em Python 3/2923.py
while True:
    try:
        vet=input().split()
        t=int(vet[0])
        a=int(vet[1])
        w=int(vet[2])
        c=int(vet[3])
        veri =(a*100)/t
        if veri>=w and veri<c:
            print("warning")
        elif veri>=c:
            print("critical")
        else:
            print("OK")
    except EOFError:
        break